src/install/linux.md

```diff
@@ -9,14 +9,14 @@ js:
   defer: true
 ---
 
-If you're using Debian/Ubuntu, you can choose two options,
+If you're using Debian/Ubuntu on amd64 (64-bit Intel), you can choose two options,
```

Comment:
From a brief web search (e.g. http://www.x86-64.org/about.html), it looks like amd64 should be AMD64.